app/models/manageiq/providers/cloud_manager/vm.rb

```diff
@@ -26,6 +26,8 @@ class ManageIQ::Providers::CloudManager::Vm < ::Vm
 
   has_many   :host_aggregates,        :through => :host
 
+  belongs_to  :server_group
```

Comment:
If you're defining this in the base vm_or_template class you don't need it here in the CloudManager::Vm subclass as well

Reply:
have removed it from here.